src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,58 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final String NULL_MSG = "%s can`t be null";
+    public static final String LESS_MSG = "%s less than minimum: expected %d but was %d!";
+    public static final String EXIST_MSG = "A user with this email already exists!";
+    public static final String LOGIN = "Login";
+    public static final String PASSWORD = "Password";
+    public static final String AGE = "Age";
+    public static final int USER_DATA_MIN_LENGTH = 6;
+    public static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationException("User can`t be null!");
```

Comment:
If you make all the messages constants, I think it would be better to add this one as well

Reply:
Other messages are repetitive; this one is not.